devicetypes/sky-nie/minoston-door-window-sensor.src/minoston-door-window-sensor.groovy

```diff
@@ -0,0 +1,674 @@
+/**
+ *     Minoston Door/Window Sensor v1.0.3
+ *
+ *  	Models: MSE30Z
+ *
+ *  Author:
+ *   winnie (sky-nie)
+ *
+ *	Documentation:
+ *
+ *  Changelog:
+ *
+ *    1.0.3 (07/16/2021)
+ *     - change lastBatteryReport to record the time of fresh battery
+ *     - add lastBattery to record the battery value
+ *
+ *    1.0.2 (07/15/2021)
+ *      - update ConfigParams as product designed
+ *      - update DTH name as product designed
+ *
+ *    1.0.1 (07/13/2021)
+ *      - Syntax format compliance adjustment
+ *      - delete dummy code
+ *
+ *    1.0.0 (04/26/2021)
+ *      - Initial Release
+ *
+ * Reference：
+ *   https://community.smartthings.com/t/release-aeotec-trisensor/140556?u=krlaframboise
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition(name: "Minoston Door/Window Sensor", namespace: "sky-nie", author: "winnie", ocfDeviceType: "x.com.st.d.sensor.contact", executeCommandsLocally: false, genericHandler: "Z-Wave") {
```

Comment:
@Winniewen18 Why did You add another DTH to this pull request ? That should be in a separate CST request.

Reply:
Oh, Something similar happened before.This is caused by upgrading and modifying the name.Not another DTH:
  
![1626420119_副本](https://user-images.githubusercontent.com/54890556/125908673-35f391dd-ac5e-42b4-9a2a-2cb7c7c7bcec.jpg)
   Because the products related to the brand “Evalogik” have been suspended, this DTH has nothing to do with  “Evalogik”. In order not to cause confusion and misunderstanding, it was renamed.

Reply:
@sky-nie that's not just renaming the current DTH. 
Another code (completely different DTH) was submitted. Please leave in this PR only the DTH that was reviewed and rename it if it's needed.

Reply:
@KKlimczukS :
   Before I renamed "Evalogik Door/Window Sensor"  to "Minoston Door/Window Sensor" to this DTH, There was another DTH with name of "Minoston Door/Window Sensor" in my fork(which is not related with Minoston Door/Window Sensor ).  I deleted it before the rename action. 
   What can I do now? restore the current DTH's name?

Reply:
@sky-nie 
Can You try to 
- revert DTH's (the one that was reviewed) name to the previous one and commit
- delete DTH (the one that is unnecessary) and commit
- change the name of the DTH again and commit

Reply:
I have completed the first step. The second step,,, now I can not see the unnecessary DTH. Is the state back to normal? The third step——I think renaming should not be a necessary action, it should be ok to add enough comments in the file.

Reply:
This unnecessary DTH is still here in this PR, so You need to have Your local branch and this PR synchronized.

Reply:
I have  sync the local branch with my fork, update the locale branch , then commit  the locale branch to [this PR](https://github.com/sky-nie/SmartThingsPublic/tree/devws-for-nie-tech-co-ltd-contai_51);

Reply:
@sky-nie Please create a fresh branch and issue new pull request (and add a link to it in this PR).

Reply:
@KKlimczukS :
   I have create a fresh branch and issue new pull request( [the PR](https://github.com/sky-nie/SmartThingsPublic/tree/devws-for-nie-tech-co-ltd-contai_53))

Reply:
@KKlimczukS 
    I have start [a new PR](https://github.com/sky-nie/SmartThingsPublic/tree/devws-for-nie-tech-co-ltd-contai_53) for the project. Please review the PR and closed this PR;